### PR TITLE
Docker Compose support

### PR DESCRIPTION
**Pull request by ryanrolds**
Watcher support Windows required using Webpack's poller.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM node:12-alpine
+
+RUN apk add --no-cache mariadb-dev make g++ linux-headers nodejs tzdata
```
Comment (csjoblom) [inline Dockerfile]: Do we need mariadb as part of this?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+WORKDIR /website
+
+COPY package.json .
+COPY package-lock.json .
+
+RUN npm install
+RUN npm install nuxt --global
```
Comment (csjoblom) [inline Dockerfile]: Do we want to specify the nuxt version here?

**Review by csjoblom — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ryanrolds) [reply Dockerfile]: Derp, copy paste job. I will fix.

File: Dockerfile (hunk 2 above)
Comment (ryanrolds) [reply Dockerfile]: Ah good point.

**Review by csjoblom — APPROVED** [review]
👍 looks good thanks @ryanrolds

**Pull request merged**